google/cloud/bigquery/v2/minimal/internal/job_request_test.cc

```diff
@@ -167,8 +163,10 @@ TEST(ListJobsRequestTest, Success) {
       "https://bigquery.googleapis.com/bigquery/v2/projects/1/jobs");
   expected.AddQueryParameter("allUsers", "true");
   expected.AddQueryParameter("maxResults", "10");
-  expected.AddQueryParameter("minCreationTime", std::to_string(kMinInt));
-  expected.AddQueryParameter("maxCreationTime", std::to_string(kMaxInt));
+  expected.AddQueryParameter("minCreationTime",
+                             std::to_string(kMinCreationTime));
+  expected.AddQueryParameter("maxCreationTime",
+                             std::to_string(kMaxCreationTime));
```

Comment:
You can choose, but I think I would recommend just...
```
auto const kMinCreationTime = std::chrono::system_clock::from_time_t(0) + 
                              std::chrono::milliseconds{1111111111111};
auto const kMaxCreationTime = std::chrono::system_clock::from_time_t(0) +
                              std::chrono::milliseconds{1111111111211};
```
and then here using, for example,...
```
    std::to_string(TimePointToUnixMilliseconds(kMinCreationTime))
```